### ISSUE TITLE
Fail to install {paradox}: "unimplemented STRING_NO_NA"

### ISSUE THREAD

**Issue by pat-s** [oracle/fastr#156]
I've found no match for the error message and hence assume it's a new one.

```r
> install.packages("paradox")
Content type 'application/octet-stream' length 40872 bytes (39 KB)
setrlimit to increase file descriptor limit failed, errno 22
* installing *source* package ‘paradox’ ...
** package ‘paradox’ successfully unpacked and MD5 sums checked
** using staged installation
** R
** byte-compile and prepare package for lazy loading
setrlimit to increase file descriptor limit failed, errno 22
unimplemented STRING_NO_NA
ERROR: lazy loading failed for package ‘paradox’


FastR version 3.6.1 (2019-07-05)
Platform: x86_64-apple-darwin19.5.0 (64-bit)
Running under: macOS Catalina 10.15.5

Matrix products: default
BLAS:   /Library/Java/JavaVirtualMachines/graalvm-ce-java11-20.1.0/Contents/Home/languages/R/lib/libRblas.dylib
LAPACK: /Library/Java/JavaVirtualMachines/graalvm-ce-java11-20.1.0/Contents/Home/languages/R/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base

loaded via a namespace (and not attached):
[1] tools_3.6.1

```

- GraalVM 20.1.0

**Comment by steve-s** [comment]
Hello Patrick,

thank you for the report. Yes we're missing this function `STRING_NO_NA` (and bunch or similar ones like `REAL_NO_NA`). We'll add them.

**Comment by pat-s** [comment]
Thanks! I'll have time to wait until this is supported.

**Comment by steve-s** [comment]
Fixed in 4bd8f5c7076fcb2a36623ed1f68e8c02fdaade2d
